### DISCUSSION TITLE
CTU Prague Xmas video featuring iCub

### DISCUSSION THREAD

**Discussion by matejhof** [robotology/community#565] [Show and tell]
Hello @robotology/everyone,
I hope you will enjoy this robot Xmas video from Prague! ;)  
https://youtu.be/wiE7GfN358s
cheers,
Matej

**Comment by Uboldi80** [comment]
The video is Beautiful!!!

**Comment by pattacini** [comment]
Amazing Xmas video, really!
I'm writing the letter to Santa Claus to get the Johnny 5 lego robot 🎅🏻

**Comment by alecive** [comment]
Super cool!

**Comment by alexbernardino** [comment]
Merry Christmas, everybody!
AB

Alessandro Roncone ***@***.***> escreveu no dia quarta,
1/12/2021 à(s) 22:29:

**Comment by claudiofantacci** [comment]
AHAHAH that's indeed a cool video!
🎅 🤖

**Comment by lornat75** [comment]
nice!
